Update getting started guide to work on any Kubernetes cluster (#147)
scalar-labs/helm-charts: docs/getting-started-monitoring.md

# Getting Started with Helm Charts (Monitoring using Prometheus Operator)

> This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing. And, we assume that you can use any Kubernetes cluster, but we use **Minikube** in this document.

Reviewer: ```suggestion
This document explains how to get started with Scalar products monitoring on Kubernetes using Prometheus Operator (kube-prometheus-stack). Here, we assume that you already have a Mac or Linux environment for testing. We use **Minikube** in this document, but the steps we will show should work in any Kubernetes cluster.
```
